src/HealthGPS/analysis_module.cpp

```diff
@@ -531,6 +583,26 @@ void AnalysisModule::classify_weight(DataSeries &series, const Person &entity) c
     }
 }
 
+void AnalysisModule::classify_weight(const Person &person) {
+    auto weight_class = weight_classifier_.classify_weight(person);
+    switch (weight_class) {
+    case WeightCategory::normal:
+        calculated_stats_[get_channel_index("normal_weight")]++;
+        break;
+    case WeightCategory::overweight:
+        calculated_stats_[get_channel_index("over_weight")]++;
+        calculated_stats_[get_channel_index("above_weight")]++;
```

Comment:
What's the difference between `over_weight` and `above_weight`?

Reply:
No idea. They're both in the existing analysis module code though. I guess `above_weight` was added at some point and has some use downstream.

Reply:
One includes everyone in  overweight and obese, and the other just overweight; I forget which is which. TBH the accumulated group is redundant and can be removed, since you can get it by summing the above weight categories anyway.

Reply:
That's a really confusing naming scheme, though I guess if we change it that will break things :worried: